backend/frontend/templates/frontend/page_course_component_details/base.html

```diff
@@ -0,0 +1,20 @@
+{% extends "frontend/base.html" %}
```

Comment:
Should we not be extending base_logged_in? Only logged in users should be able to see this.